spark/src/main/kotlin/com/adevinta/spark/components/buttons/Button.kt

```diff
@@ -117,7 +117,10 @@ internal fun BaseSparkButton(
             Spacer(Modifier.width(SparkButtonDefaults.IconSpacing))
         }
         ProvideTextStyle(value = SparkTheme.typography.callout) {
-            content()
+            Row(
+                modifier = Modifier.weight(1f, false),
```

Comment:
nit: can we add an explicit parameter name here?

Reply:
Sure!